doc/man3/OSSL_sleep.pod

```diff
@@ -0,0 +1,37 @@
+=pod
+
+=head1 NAME
+
+OSSL_sleep - delay execution for a specified amount of milliseconds
+
+=head1 SYNOPSIS
+
+ #include <openssl/crypto.h>
+
+ void OSSL_sleep(unsigned long millis);
+
+=head1 DESCRIPTION
+
+OSSL_sleep() is a convenience function to delay execution of the calling
+thread for (at least) I<millis> milliseconds.  The delay may be lengthened
+slightly by system activity, by the time spent processing the call or by
```

Comment:
d/slightly/

Reply:
I disagree.  Without it, it could arguably mean that two seconds extra would be considered perfectly normal...  I'd call that surprising.  I do think "slightly" has significance here

Reply:
It is entirely possible that the OS adds two seconds, ten seconds or even a minute.  It depends what else is going on.  _Slightly_ is over promising IMO.

Reply:
I'll also note that with out current implementation, it is entirely possible for the call to return **before** the specified time.  `nanosleep(2)` + signal e.g.

Reply:
Maybe that part should be changed to something like "The delay is not guaranteed; it may be affected by system activities, by time spent processing the call ..."

Reply:
Change done

Reply:
s/slightly/according to OS internal behaviour/ ?